src/dyadicop.jl

```diff
@@ -35,7 +35,7 @@ end
 
 function assemble!(biop::DyadicOp, tfs::Space, bfs::Space, store,
     threading::Type{BEAST.Threading{:multi}};
-    quadstrat=defaultquadstrat(biop, tfs, bfs))
+    kwargs...)
```

Comment:
Why is this better?

Reply:
If we want further in the assemble! tree to add an argument, even for assembling one specific operator, we dont have to change all assemble functions. I think in a tree it is a good practise to always pas the kwargs argument, enabling us to add extra arguments for specific functions. in the leaves a kwargs argument is needed to catch unused argements intended for other leaves